Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift

```diff
@@ -61,14 +61,14 @@ struct AnimationsEnvironment {
 
 let animationsReducer = Reducer<AnimationsState, AnimationsAction, AnimationsEnvironment> {
   state, action, environment in
-  enum CancelID {}
+  enum CancelId {}
```

Comment:
We've recently changed these from `Id` to `ID` on the `concurrency-beta` branch, which will be merged soon. Care to revert these changes?
```suggestion
  enum CancelID {}
```

Reply:
oh, I didn't catch that branch, sorry.
I confirmed you already reverted `CancelId` → `CancelID`. Thanks!